cli/js/compiler.ts

```diff
@@ -640,8 +644,7 @@ function buildLocalSourceFileCache(
       let mappedUrl = importDesc.resolvedSpecifier;
       const importedFile = sourceFileMap[importDesc.resolvedSpecifier];
       assert(importedFile);
-      const isJsOrJsx =
-        importedFile.mediaType === MediaType.JavaScript ||
+      const isJsOrJsx = importedFile.mediaType === MediaType.JavaScript ||
```

Comment:
I've often wondered if I should just make this go to the next line as I suspect it will probably be a lot easier to format certain scenarios...